provider/haproxy/config/rsyslogd.conf

```diff
@@ -3,15 +3,12 @@ $UDPServerRun 8514
 $template CustomFormat,"%timegenerated% %HOSTNAME% %syslogtag%%msg%\n"
 $ActionFileDefaultTemplate CustomFormat
 
-/* info */
-if $programname contains 'haproxy' and $syslogseverity == 6 then
-    action(type="omfile" file="/var/log/haproxy/traffic")
-
-if $programname contains 'haproxy' and $syslogseverity-text == 'err' then
-    action(type="omfile" file="/var/log/haproxy/errors")
-
-/* notice */
-if $programname contains 'haproxy' and $syslogseverity <= 5 then
-    action(type="omfile" file="/var/log/haproxy/events")
-
-*.* /var/log/messages
+if $programname contains 'haproxy' then {
+  if $syslogseverity == 6 then
+      action(type="omfile" file="/var/log/haproxy/traffic")
+  if $syslogseverity-text == 'err' then
```

Comment:
Oh - yes. [rsyslog properties](http://www.rsyslog.com/doc/v8-stable/configuration/properties.html) mention syslogseverity-text is just severity in numerical form. Will change. 

On that note, syslogseverity for error is 3 - should they be duplicated in /var/log/events as well?